pkg/store/sqlite.go

```diff
@@ -285,7 +285,7 @@ func listAccountsModifiedCommon(tx Queryable, cursor int64, limit int) (ids []st
 }
 
 func getInvoiceCommon(tx Queryable, addr giga.Address) (giga.Invoice, error) {
-	row := tx.QueryRow("SELECT invoice_address, account_address, txn_id, vendor, items, key_index, block_id, confirmations FROM invoice WHERE invoice_address = ?", addr)
+	row := tx.QueryRow("SELECT invoice_address, account_address, txn_id, vendor, items, key_index, block_id, confirmations, created FROM invoice WHERE invoice_address = ?", addr)
```

Comment:
These changes were already merged in your previous PR, which means you haven't pulled Main branch before creating this branch :)  Lines 288, 298, 319.

It might merge OK, or might not...